### PR TITLE
Add Debian 9 support

### PR DESCRIPTION
**Pull request by bastelfreak**
This is a rebase of https://github.com/voxpupuli/puppet-nginx/pull/1154

### CODE REVIEW

File: spec/acceptance/class_spec.rb [hunk 1]
```diff
@@ -42,17 +42,21 @@ class { 'nginx':
...
         apply_manifest(pp, catch_failures: true)
+        apply_manifest(pp, catch_changes: true)
       end
 
       describe package('nginx') do
         it { is_expected.to be_installed }
-        it 'comes from the expected source' do
-          pkg_output = shell(pkg_cmd)
-          expect(pkg_output.stdout).to match pkg_match
+        # the passenger people package ngin, except for Debian 9 systems!
```
Comment (ekohl) [inline spec/acceptance/class_spec.rb]: s/ngin/nginx/

File: spec/acceptance/class_spec.rb [hunk 2]
```diff
@@ -42,17 +42,21 @@ class { 'nginx':
...
+        unless fact('os.family') == 'Debian' && fact('os.release.major') == '9'
```
Comment (ekohl) [inline spec/acceptance/class_spec.rb]: I'd prefer to change ```pkg_match``` above on Debian 9 to verify it comes from Debian itself. That makes this testing a bit more straightforward.

File: spec/acceptance/class_spec.rb (hunk 1 above)
Comment (bastelfreak) [reply spec/acceptance/class_spec.rb]: fixed

File: spec/acceptance/class_spec.rb (hunk 2 above)
Comment (bastelfreak) [reply spec/acceptance/class_spec.rb]: sounds good, I will update it.

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**